lms/static/scripts/frontend_apps/components/dashboard/test/DashboardFooter-test.js

```diff
@@ -0,0 +1,17 @@
+import { checkAccessibility } from '@hypothesis/frontend-testing';
+import { mount } from 'enzyme';
+
+import DashboardFooter from '../DashboardFooter';
+
+describe('DashboardFooter', () => {
+  function createComponent() {
+    return mount(<DashboardFooter />);
+  }
+
+  it(
+    'should pass a11y checks',
+    checkAccessibility({
+      content: () => createComponent(),
+    }),
+  );
```

Comment:
I have just added accessibility checks, because the footer is a static presentational component.